src/core/selinux-access.c

```diff
@@ -163,6 +164,29 @@ static int access_init(sd_bus_error *error) {
         return 1;
 }
 
+int mac_selinux_callback_check(
+                const char *name,
+                void *userdata) {
+
+        _cleanup_free_ char *path = NULL;
+        struct mac_selinux_callback_userdata *t = userdata;
+        int exists;
+
+        assert(name);
+        assert(t);
+
+        if(!mac_selinux_use())
```

Comment:
space between the `if` and the initial `(`. See CODING_STYLE

Reply:
done